### PR TITLE
add min and max worker params to `cluster.adapt` docs

### PR DESCRIPTION
**Pull request by jgerardsimcock**
This pull request is a documentation update prompted from this https://github.com/dask/dask-kubernetes/issues/62.

### CODE REVIEW

File: doc/source/conf.py [hunk 1]
```diff
@@ -40,7 +40,7 @@
     'sphinx.ext.autosummary',
     'sphinx.ext.extlinks',
     'sphinx.ext.intersphinx',
-    'numpydoc',
```
Comment (mrocklin) [inline doc/source/conf.py]: I'm curious, what was the motivation for this change?

File: doc/source/index.rst [hunk 2]
```diff
@@ -22,7 +22,7 @@ Quickstart
    cluster = KubeCluster.from_yaml('worker-spec.yml')
    cluster.scale_up(10)  # specify number of nodes explicitly
 
-   cluster.adapt()  # or dynamically scale based on current workload
+   cluster.adapt(minimum=1, maximum=100)  # or dynamically scale based on current workload
```
Comment (mrocklin) [inline doc/source/index.rst]: +1 from me on this.  Also see https://github.com/dask/dask/pull/3371

File: doc/source/conf.py (hunk 1 above)
Comment (jgerardsimcock) [reply doc/source/conf.py]: Ah yes. Sorry. I was unable to build the docs locally without modifying the config file. 

screen shot 2018-04-03 at 1 02 17 pm

Comment (mrocklin) [reply doc/source/conf.py]: See the `doc/requirements-docs.txt` file.  From the error above it looks like you need to `pip install numpydoc`.

If you revert this change I'll be happy to merge.  Moving off of numpydoc would be a larger conversation than the doc change below.

**Comment by mrocklin** [comment]
Thanks for taking the time to improve this @jgerardsimcock !

**Pull request merged**